src/ccmodules/package_upgrade.c

```diff
@@ -62,7 +62,7 @@ void package_upgrade_handler(GNode *node) {
 #elif defined(PACKAGE_MANAGER_YUM)
 		exec_task("/usr/bin/yum update");
 #elif defined(PACKAGE_MANAGER_DNF)
-		exec_task("/usr/bin/dnf update --refresh");
+		exec_task("/usr/bin/tdnf update --refresh");
```

Comment:
but, I did not install tdnf in /usr/bin/   u_u
a solution for this could be to add something like this [configure.ac](https://github.com/clearlinux/clr-cloud-init/blob/master/configure.ac#L75)  ?